.gitignore

```diff
@@ -18,5 +18,28 @@ responses/
 # virtualenv
 venv/
 
-# editors
-.vscode/
```

Comment:
I think we should keep that vscode config folder because it contains stuff that is relevant for every developer. Anything really user-specific can go to the user settings, outside of the project.